libsolidity/lsp/LanguageServer.cpp

```diff
@@ -101,14 +101,29 @@ Json::Value LanguageServer::toJson(SourceLocation const& _location)
 void LanguageServer::changeConfiguration(Json::Value const& _settings)
 {
 	m_settingsObject = _settings;
+	Json::Value jsonIncludePaths = _settings["include-paths"];
+	if (jsonIncludePaths && jsonIncludePaths.isArray())
+	{
+		std::vector<boost::filesystem::path> includePaths;
+		for (Json::Value jsonPath: jsonIncludePaths)
+		{
+			if (!jsonPath.isString())
+			{
+				m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
```

Comment:
Should this be maybe a proper error with throwing and all?

Reply:
Not with throwing, at most reporting (trace does report towards the client), as this is an error caused by the client.

Reply:
Then maybe reporting it at a higher log level tahn trace?